### DISCUSSION TITLE
Does Postal support DANE?

### DISCUSSION THREAD

**Discussion by JohnHammell** [postalserver/postal#3151] [Help with using Postal]
Does Postal support DANE?

I asked about MTA-STS yesterday which Postal appears to not yet support. In my readings about MTA-STS, I've read that MTA-STS is basically the silver standard and that DANE is the gold standard for what it does.

So, with that, I'm wondering if Postal supports DANE.

Either or both DANE & MTA-STS seem they would be good additions to Postal.

**Comment by willpower232** [comment]
You get the honour of being the first to raise DANE as a feature request I guess :sweat_smile: 

I don't believe it is currently supported but also looks a little complicated so unless someone else does the work, I am not sure it would be a priority for the developers.
